Cargo.toml

```diff
@@ -194,4 +194,4 @@ revm-interpreter = { git = "https://github.com/bluealloy/revm", branch = "reth_f
 revm-precompile = { git = "https://github.com/bluealloy/revm", branch = "reth_freeze" }
 revm-primitives = { git = "https://github.com/bluealloy/revm", branch = "reth_freeze" }
 
-foundry-compilers = { git  = "https://github.com/Deniallugo/compilers.git", branch = "main", default-features = false }
+foundry-compilers = { git  = "https://github.com/foundry-rs/compilers", branch = "main", default-features = false }
```

Comment:
Are you sure it's also just the main branch in the main repository

Reply:
I will update to use a commit hash instead of branch.